style.css

```diff
@@ -31,14 +31,61 @@
     display: none;
   }
 
-  .head-top a {
+  .mobile-nav {
+    position: inherit;
+    display: block;
+  }
+
+  .x-btn {
+    position: absolute;
+    right: 10%;
+    color: white;
+    font-size: 2.5rem;
+    z-index: 99;
+  }
+
+  .display-class {
+    display: none;
+  }
+
+  .bg-style {
+    background-color: rgba(96, 112, 255, 0.7);
+    backdrop-filter: blur(8px);
+    width: 100vw;
+    height: 100vh;
+    position: absolute;
+  }
```

Comment:
The menu is supposed to be an overlay and not scrollable, however, your mobile nav menu is not fixed hence can be scrolled and dissappear from the screen. This does not give a good user experience. Kindly ensure that you fix this to match the figma design and achieve standard best practices. :wink: 

### Your design when scrolled

![Screenshot from 2023-01-03 15-54-52](https://user-images.githubusercontent.com/65233860/210395549-c0bbd57c-67c5-44e2-b854-35099ba37d2c.png)


### Figma design 

![Screenshot from 2023-01-03 15-54-18](https://user-images.githubusercontent.com/65233860/210395584-4584dc4e-0c0d-42db-b8de-bc763bd5bdf6.png)